staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/unstructured.go

```diff
@@ -450,10 +450,14 @@ func (u *Unstructured) SetFinalizers(finalizers []string) {
 }
 
 func (u *Unstructured) GetManagedFields() []metav1.ManagedFieldsEntry {
-	items, found, err := NestedSlice(u.Object, "metadata", "managedFields")
```

Comment:
I see, so this is NestedSlice without the deepcopy

Reply:
All the typed `Nested*` funcs return copies. Only `NestedFieldNoCopy` does not.

We could have `NestedSliceNoCopy` and `NestedMapNoCopy` because they only type-cast. For some reason we chose not to. Interestingly we have `nestedMapNoCopy` but don't export it.